config/packages/remote-phase/package-operator-remote-phase-manager.Deployment.yaml.gotmpl

```diff
@@ -6,6 +6,7 @@ metadata:
   creationTimestamp: null
   labels:
     app.kubernetes.io/name: package-operator-remote-phase-manager
+    hypershift.openshift.io/need-management-kas-access: "true"
```

Comment:
I think we only need the label in the pod template and not on the deployment itself. The issue is worded ambiguously though

Reply:
Yup - Example: https://github.com/openshift/hypershift/blob/4072623a71089bb242d0db772db296a191ff720a/control-plane-operator/controllers/hostedcontrolplane/snapshotcontroller/testdata/zz_fixture_TestReconcileOperatorDeployment.yaml#L31